local/bin/py/build/configurations/pull_config_preview.yaml

```
@@ -49,7 +49,7 @@
     contents:
 
     - action: integrations
-      branch: prod
+      branch: ruth/docs-2083
```

```suggestion
      branch: prod
```
This is a temporary change for the preview site.